src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -412,7 +412,8 @@ def show(
         view_2d: Dict = None,
         name_filter: str = None,
         dark_mode: bool = False,
-        **plotting_options,
+        plotting_options: Optional[Dict[str, Any]] = None,
+        **show_options: Any,
```

Comment:
```suggestion
        **show_options: Dict[str, Any],
```
Just wondering if you should add this type hint instead ? Do you expect `show_options` to be using something else than a dict ?